backend/package-lock.json

```diff
@@ -8198,6 +8198,5039 @@
         "xtend": "^4.0.0"
       }
     },
+    "postman-templating": {
```

Comment:
As mentioned, I find it weird to have so many of the module's dependency installed in the package-lock. I don't quite understand how package-lock works and why it is like this, will probably spend more time reading up.

Reply:
I've tried running `npm dedupe` and it managed to reduce the number of lines changed to slightly less than 2000. From my understanding, what it does is that it tries to surface the common dependencies higher in dependency tree. Can take a look at [npm-dedupe](https://docs.npmjs.com/cli/dedupe).

Reply:
As a side note, most of the babel-related dependencies introduced seems to be coming from `jest`. There's an issue on their repo about ongoing work to reduce the number/size of their dependencies.